queries/tornado_session.py

```diff
@@ -464,6 +468,18 @@ def _poll_connection(self, fd):
                 self._futures[fd].set_exception(
                     psycopg2.OperationalError('Connection error (%s)' % error)
                 )
+        except psycopg2.OperationalError as error:
```

Comment:
This block feels heavy-handed. There's no other cleanup/close function already that would do a majority of the cleanup?

Reply:
@gmr [_exec_cleanup()](https://github.com/cknave/queries/blob/f5998679f7db492ebcfe781c1d01ef8c3ce5e6af/queries/tornado_session.py#L411) does too much, but I could split out the connection-related parts into another function.